src/components/search/curated/typescript/index.tsx

```diff
@@ -69,4 +73,50 @@ Love them or hate them, static types are here to stay, and at the very least an
   )
 }
 
+const CourseFeatureCard = ({resource, className}: any) => {
```

Comment:
suggestion: I think there's a strong case for just adding these components to a folder for the courses so we have a library of "feature" cards for these resources that we can use across the site. 🤔